speech.yaml

```diff
@@ -226,32 +238,75 @@ components:
           maxLength: 10240
         last_name:
           type: string
-          description: The last name of the customer
+          description: The last name of the customer.
           maxLength: 10240
         email:
           type: string
           format: email
-          description: The email address of the customer
+          description: The email address of the customer.
         phone_number:
           type: string
           description:
             The phone number of the customer, typically from
-            ANI (Automatic Number Identification)
+            ANI (Automatic Number Identification).
           maxLength: 10240
         connection_id:
+          deprecated: true
           type: string
-          description:  The unique identifier of a specific connection profile
+          description:
+            The unique identifier of a specific connection profile.  This value
+            does not typically need to be set.  Please consult with your Medallia
+            Speech specialist to determine if a value is required.  This field has
+            been deprecated since Medallia Experience Cloud version 22CR3.0, please
+            use `connector_id` instead.
           maxLength: 10240
         profile_uuid:
+          deprecated: true
           type: string
           description:
-            The unique identifier (UUID) of the customer profile
-            (will be deprecated soon, use `connection_id` instead)
+            This field has been deprecated since Medallia Experience Cloud
+            version 21CR3.0, please use `connection_id` instead.
           maxLength: 10240
+        engine:
+          type: string
+          description:
+            The Medallia Speech engine to use to perform the
+            transcription processing.  The value to use is determined by
+            your Medallia Speech specialist during the initial audio
+            analysis process while onboarding to Medallia Speech.
+          default: "Engine 1"
+          enum:
+            - "Engine 1"
+            - "Engine 2"
+            - "Engine 3"
+        connector_id:
+          type: string
+          format: uuid
+          description:
+            The unique identifier of a specific Speech Connector API as configured
+            in Medallia Admin Suite.  If set, the Speech data in the API request is
+            routed through the Connector for processing, including for extra
+            metadata available in `speech_additional_info`.  This field was
+            introduced with Medallia Experience Cloud 22CR3.0.
+        speech_additional_info:
+          type: object
+          description:
+            Additional metadata to add to the Medallia Speech record.  Mapping this
+            data to the record is performed in setting up the Medallia Speech API
+            Connector in Admin Suite.  Use of this field requires setting a
+            `connector_id`.
+          additionalProperties:
```

Comment:
We should be able to remove this additionalProperties field as `speech_additional_info` is already type object and supports any type of valid json schema (nested objects, array, all types, etc)